### ISSUE TITLE
Feature: cumuluspage tag

### ISSUE THREAD

**Issue by chrisvidal** [initbiz/cumuluscore-plugin#49]
it would be cool to have a twig tag `cumuluspage` to create link directly to cumuluspage instead of doing it with `|page{'cluster': cluster.username}` 
or did I miss it?

**Comment by tomaszstrojny** [comment]
Hi @chrisvidal,

I agree it may be a good idea but not without problems. In general, you can name the `:cluster` parameter differently in your pages and everything should still work. We wouldn't be able to detect its name in the filter. What is more, you can use cluster's slug in some cases, and not username. Not sure we could do something much easier than the typical `|page{'cluster': cluster.username`.  If you have any ideas, respond to this issue.

Cumulus gives integration with `RainLab.Pages` plugin and registers custom nav items type (cumulus page). They can be automatically hidden basing on features selected and they will have the cluster parameter filled if it's the same for all pages. It's not perfect, too, but could save some time and make better UX for admins.

Tomasz

**Comment by chrisvidal** [comment]
the issue I am facing is that I created an admin menu under rainlab pages.
each menu item is attached to a cumuluspage target a page with multiple parameters on top of `:cluster`, for instance `/app/:cluster/clients/:mode/:id` as I created my component acting as controller handling the all CRUD experience in one page. 
But, when I am navigating down the whole link, for instance `/app/client-username/clients/edit/128`, the menu items URL inherit the params `:mode` and `:id`, hence ht menu item do not work as it links to the same page instead of the parent page. It is probably a default behavior from rainlab page, I am not sure but I was thinking that if a twig filter `cumuluspage` edited it could handle the link to the page. But you are right, the filter wouldn't know the name of the param holding the slug or the username.
